code/src/terrat_files/github/sql/select_repo_tree.sql

```diff
@@ -1,5 +1,24 @@
+with
+rt1 as (
+    select * from repo_trees where installation_id = $installation_id and sha = $sha
```

Comment:
Any reason why we are not restricting the amount of fields selected here just to `id, path, changed`?

Reply:
In this case we're really just making aliases to `repo_trees` given a filter, so I chose to do `*` because:

1. Since it's a CTE, it doesn't actually cost anything to select all the data (as far as I understand CTEs, that is).
2. It means any query that uses it gets access to all the data.

Mainly because it's a CTE, there shouldn't be any cost in selecting all.  But let me know If I'm wrong on this.

Reply:
I think you are right for CTEs. 

I tend to restrict my selects (as a policy) because on JOINs it usually means that pg will fetch the heap even when it actually uses an index.

Reply:
Its ok then